src/Generator/Controller/Generator.php

```diff
@@ -55,11 +56,14 @@ public function rules(): array
             [
                 'controllerClass' => [
                     new Required(),
-                    (new MatchRegularExpression('/^[\w\\\\]*Controller$/'))
-                        ->message('Only word characters and backslashes are allowed, and the class name must end with "Controller".')
+                    (new MatchRegularExpression('/^[\w]*Controller$/'))
```

Comment:
How about to check that the first letter is upper case?

Reply:
done